apps/docs/src/components/LeftSidebar/LeftSidebar.astro

```diff
@@ -36,10 +36,14 @@ const sidebar = SIDEBAR[langCode];
                       <div class="subTitle flex items-center cursor-pointer text-sm m-[1px] py-[0.3rem] px-2 text-inherit hover:no-underline focus:no-underline focus-visible:no-underline hover:bg-theme-gray-950 dark:hover:bg-theme-gray-400 focus-visible:bg-theme-gray-950 dark:focus-visible:bg-theme-gray-400 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-4 focus-visible:outline-theme-gray-100 dark:focus-visible:outline-white">
                         <a>{child.text}</a>
                         <div class="caret-icon z-[1]">
-                          <CaretDownIcon width={24} height={24} />
+                          <CaretDownIcon
+                            width={24}
+                            height={24}
+                            class="stroke-black dark:stroke-white"
+                          />
                         </div>
                       </div>
-                      <ul class={`${shouldBeExpanded ? '' : 'hidden'} submenu`}>
+                      <ul class={`${!shouldBeExpanded && 'hidden'} submenu`}>
```

Comment:
that change will inline `false` or `undefined` string as a class

Reply:
Actually, I don't think the returned value from this code could equal `undefined`, here is a screenshot
<img width="191" alt="image" src="https://user-images.githubusercontent.com/77807844/228756213-1708e5fb-3a26-4efe-9910-01fb59d9aa0e.png">
If I'm wrong, explain what logical conjunction could return that, please 😄
But with return value: `false`, you're right!